Source/Core/Core/Analytics.cpp

```diff
@@ -205,14 +335,20 @@ void DolphinAnalytics::ReportPerformanceInfo(PerformanceSample&& sample)
   }
 }
 
+u64 DolphinAnalytics::GetSamplingJitterMicroseconds() const
```

Comment:
Given the refactoring, maybe make use of `std::chrono::microseconds`, which should eliminate the need for conversion code, since the API handles that.

Reply:
Done.